mmv1/templates/terraform/examples/alloydb_cluster_full.tf.erb

```diff
@@ -0,0 +1,45 @@
+resource "google_alloydb_cluster" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
It looks like this is not exercising a few fields - would it be possible to add these fields to the test? (Or if not, why not?)

- annotations
- display_name
- initial_user.user
- ssl_config
- automated_backup_policy.weekly_schedule.start_times.minutes (and seconds and nanos)


I see that timeBasedRetention is mutually exclusive with quantityBasedRetention so it has to be skipped 👍

Reply:
did a test. will add a few missing ones, but...

annotations
ssl_config

are not returned back from api so it causes diffs

Reply:
interesting! Those fields will need to be handled somehow so that they don't cause diffs.

SSL config: this seems like a fairly important field; maybe it would make sense to do a similar custom flattener?
Annotations: it seems like this may not be a useful field if there's no way to retrieve the data - maybe it would make sense to leave it off of the resource?

What do you think?

Reply:
Hmm - related question - can annotations and ssl_config be updated? Or are they actually input-only?

Reply:
added initial_user.user and automated_backup_policy.weekly_schedule.start_times.minutes (and seconds and nanos) but all others (above + display_name) are not returned from api.
looking at UI/gcloud these fields are not configureable. maybe i should omit them?

Reply:
yeah if the console / gcloud don't let you set them, omitting them makes sense to me. We can always add them in later as needed / when it's possible.

Reply:
resolved in 67d5cc2810526cb2dbd1c03aaa7307a0a1a04d1b